### DISCUSSION TITLE
Unexpected Behavior for Tabset Headers With Emoji

### DISCUSSION THREAD

**Discussion by fnafox** [requarks/wiki#6776] [Error / Bug Report]
# Environment
Wiki.js 2.5.300
Node.js 16.20.2
11.1.2-MariaDB
x86_64 GNU/Linux 6.4.16-1

# Issue
When using emoji (or image) in headers of a tabset, clicking on the emoji throws you back to the first tab rather than the intended one. Additionally, clicking on the emoji will not swap to the expected tab.

# Reproduce
Create a tabset such as the following, click on the Pear tab by clicking on the emoji.
Switch to the Pear tab by clicking the text, then click on the pear emoji.
```
# {.tabset}
## :apple: Apple
Quae sit provident vel. Aut fugiat odit qui. Rerum aut vel repellendus sit asperiores. Ea et asperiores deserunt nulla beatae voluptatem voluptas. Accusantium est magnam doloribus et cum non optio maxime.
## :pear: Pear
Eligendi sint nam ab quisquam nobis. Culpa optio officia tempore tenetur accusamus. Omnis consequatur omnis omnis ut. Qui reiciendis tempore est consequatur error. Ut perspiciatis optio consequatur tenetur saepe.
```
# Expected behavior
Clicking on any part of the tab should switch to the appropriate tab's content.

**Comment by rtrexmass** [comment]
I have the same behavior, but without the emoji on the header, just with text, if i click on the header "title" it sends me back to the first tab header, if i click outside the text, everything works ok.

![bugontabset](https://github.com/requarks/wiki/assets/105231805/d7345efb-139d-4c10-97a6-3f4a356f5cdb)

The highlighted area, is the area that isn't working as expected.

**Comment by itruzrp-wq** [comment]
The same problem is in the wiki documentation. As far as I understand, the 1st and 2nd tabs act as anchors for icons and bold text. I still haven't figured out how to fix this.

An example of one such tab:
`## **Step 1** (MAC address) :arrow_right:`
